### DISCUSSION TITLE
it could not play in webview or wkwebview of flutter

### DISCUSSION THREAD

**Discussion by dajun-TT** [CoderLine/alphaTab#1707] [Q&A]
qustion description
we run alphatab in webview or wkwebview of flutter, but we meet some problems.
such as: 
1. sometimes the score cannot be display.
2. if the score has been displayed, we `playPause` it. but it can not be played, and the cursor cannot run.

are there any other methods can resolve the problem? or could you have some thinkings about these problems ?  please....

**Comment by Danielku15** [comment]
Do you have any logs, error messages, screenshots, example code you can share? Without any insights its impossible to help.
